### PR TITLE
Add api docs

### PR DESCRIPTION
**Pull request by classicalliu**
- Add `apis.md` for all RPC's brief infos
- Add `poly-apis.md` for `poly_` RPCs details, including methods and types.
- Remove RPC parts from `README`

### CODE REVIEW

File: docs/poly-apis.md [hunk 1]
```diff
@@ -0,0 +1,636 @@
...
+*   `l2Sudt`: [`ScriptInfo`](#type-scriptinfo) - L2 sudt script
+
+*   `omniLock`: [`ScriptInfo`](#type-scriptinfo) - Omni lock script
+
+*   `challengeLock`: [`ScriptInfo`](#type-scriptinfo) - Challenge lock script
+
+
+### Type `EoaScripts`
+
+Supported eoa scripts
```
Comment (RetricSu) [inline docs/poly-apis.md:588]:
```suggestion
Supported eoa account lock scripts
```

File: docs/poly-apis.md [hunk 2]
```diff
@@ -0,0 +1,636 @@
...
+
+*   `meta`: [`BackendInfo`](#type-backendinfo) - Meta contract backend info
+
+*   `polyjuice`: [`BackendInfo`](#type-backendinfo) - Polyjuice backend info
+
+*   `ethAddrReg`: [`BackendInfo`](#type-backendinfo) - Eth Address Registry contract backend info
+
+
+### Type `AccountInfo`
+
```
Comment (RetricSu) [inline docs/poly-apis.md:613]:
```suggestion

Some helpful system accounts

```

File: docs/poly-apis.md [hunk 3]
```diff
@@ -0,0 +1,636 @@
...
+Describes the accounts web3 used.
+
+#### Fields
+
+`Accounts` is a JSON object with the following fields.
+
+*   `polyjuiceCreator`: [`AccountInfo`](#type-accountinfo) - Polyjuice creator account
+
+*   `ethAddrReg`: [`AccountInfo`](#type-accountinfo) - Godwoken buildin eth account
```
Comment (RetricSu) [inline docs/poly-apis.md:633]:
```suggestion
*   `ethAddrReg`: [`AccountInfo`](#type-accountinfo) - Godwoken builtin eth address mapping registry account
```

File: docs/poly-apis.md (hunk 1 above)
Comment (keroro520) [reply docs/poly-apis.md:588]: * I think "EOA" is better to be uppercase.
* s/Supported lock scripts for EOA accounts/

File: docs/poly-apis.md [hunk 4]
```diff
@@ -0,0 +1,636 @@
...
+    * [Type `Accounts`](#type-accounts)
+    * [Type `Script`](#type-scripts)
+    * [Type `ScriptHashType`](#type-scripthashtype)
+    * [Type `ScriptInfo`](#type-scriptinfo)
+    * [Type `BackendInfo`](#type-backendinfo)
+    * [Type `AccountInfo`](#type-accountinfo)
+
+## RPC Methods
+
+#### Method `poly_getCreatorId`
```
Comment (keroro520) [inline docs/poly-apis.md:35]: It is worth adding a description to every RPC method, and explaining what the returned value indicates.

**Review by keroro520 — CHANGES_REQUESTED** [review]

File: docs/poly-apis.md [hunk 5]
```diff
@@ -0,0 +1,636 @@
...
+
+Allowed kinds: "data", "type" and "data1".
+
+Refer to the section [Code Locating](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0022-transaction-structure/0022-transaction-structure.md#code-locating) and [Upgradable Script](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0022-transaction-structure/0022-transaction-structure.md#upgradable-script) in the RFC *CKB Transaction Structure*.
+
+`ScriptHashType` is equivalent to `"data" | "type" | "data1"`.
+
+*   Type "data" matches script code via cell data hash, and run the script code in v0 CKB VM.
+*   Type "type" matches script code via cell type script hash.
+*   Type "data" matches script code via cell data hash, and run the script code in v1 CKB VM.
```
Comment (keroro520) [inline docs/poly-apis.md:461]: The reference links are enough, in my opinion. We don't need to re-explain the content in the referenced documents.

Another useful RFC is https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0032-ckb-vm-version-selection/0032-ckb-vm-version-selection.md

File: docs/poly-apis.md [hunk 6]
```diff
@@ -0,0 +1,636 @@
...
+
+### Type `RollupCell`
+
+#### Fields
+
+`RollupCell` is a JSON object with the following fields.
+
+*   `typeHash`: [`H256`](#type-h256) - Hash of typeScript
+
+*   `typeScript`: [`Script`](#type-script) - Deploy info of rollup cell
```
Comment (keroro520) [inline docs/poly-apis.md:513]: It is script info, not cell info.

File: docs/poly-apis.md [hunk 7]
```diff
@@ -0,0 +1,636 @@
...
+
+
+### Type `RollupConfig`
+
+#### Fields
+
+`RollupConfig` is a JSON object with the following fields.
+
+*   `requiredStakingCapacity`: [`Uint64`](#type-uint64) - Stacking capacity
```
Comment (keroro520) [inline docs/poly-apis.md:522]:
```suggestion
*   `requiredStakingCapacity`: [`Uint64`](#type-uint64) - The minimal capacity required for staking to be the chain generator
```

File: docs/poly-apis.md [hunk 8]
```diff
@@ -0,0 +1,636 @@
...
+
+*   `challengeMaturityBlocks`: [`Uint64`](#type-uint64) - Challenge maturity blocks
+
+*   `finalityBlocks`: [`Uint64`](#type-uint64) - Finality Blocks
+
+*   `chainId`: [`Uint64`](#type-uint64) - Chain ID
```
Comment (keroro520) [inline docs/poly-apis.md:528]: A useful reference for me: https://eips.ethereum.org/EIPS/eip-155
I prefer to add this link to the type description.

File: docs/poly-apis.md [hunk 9]
```diff
@@ -0,0 +1,636 @@
...
+
+*   `gwScripts`: [`GwScripts`](#type-gwscripts) - Godwoken scripts deploy info
+
+*   `eoaScripts`: [`EoaScripts`](#type-eoaScripts) - Supported EOA scripts
+
+*   `backends`: [`Backends`](#type-backends)
+
+*   `accounts`: [`Accounts`](#type-accounts) 
+
+*   `web3ChainId`: [`Uint64`](#type-uint64) - Chain ID of web3
```
Comment (keroro520) [inline docs/poly-apis.md:554]: It should be the same with [chain id](https://github.com/nervosnetwork/godwoken-web3/pull/325/files#diff-1ccb96b1c0491eabd63696795735d87a29e86ad722db45fef58e90ed968261a7R528).

Later we can remove the prefix "web3" out, "chainId" is more clear.

File: docs/poly-apis.md [hunk 10]
```diff
@@ -0,0 +1,636 @@
...
+
+### Type `GwScripts`
+
+Godwoken scripts deploy info
+
+#### Fields
+
+`GwScripts` is a JSON object with the following fields.
+
+*   `deposit`: [`ScriptInfo`](#type-scriptinfo) - Deposit script
```
Comment (keroro520) [inline docs/poly-apis.md:565]: Deposit lock script

File: docs/poly-apis.md [hunk 11]
```diff
@@ -0,0 +1,636 @@
...
+
+*   `withdraw`: [`ScriptInfo`](#type-scriptinfo) - Withdrawal script
```
Comment (keroro520) [inline docs/poly-apis.md:567]: Withdrawal lock script

**Review by keroro520 — CHANGES_REQUESTED** [review]

File: docs/poly-apis.md [hunk 12]
```diff
@@ -0,0 +1,636 @@
...
+`EoaScripts` is a JSON object with the following fields.
+
+*   `eth`: [`ScriptInfo`](#type-scriptinfo) - Ethereum
+
+### Type `Backends`
+
+
+#### Fields
+
+`EoaScripts` is a JSON object with the following fields.
```
[the remark this answers is not in the document]
Comment (classicalliu) [reply docs/poly-apis.md]: Updated

File: docs/poly-apis.md (hunk 1 above)
Comment (classicalliu) [reply docs/poly-apis.md]: Updated

File: docs/poly-apis.md (hunk 2 above)
Comment (classicalliu) [reply docs/poly-apis.md:591]: Updated

File: docs/poly-apis.md (hunk 3 above)
Comment (classicalliu) [reply docs/poly-apis.md]: Updated

File: docs/poly-apis.md (hunk 4 above)
Comment (classicalliu) [reply docs/poly-apis.md:35]: Updated

File: docs/poly-apis.md (hunk 5 above)
Comment (classicalliu) [reply docs/poly-apis.md]: Updated

File: docs/poly-apis.md (hunk 6 above)
Comment (classicalliu) [reply docs/poly-apis.md]: Updated

File: docs/poly-apis.md (hunk 7 above)
Comment (classicalliu) [reply docs/poly-apis.md]: Updated

File: docs/poly-apis.md (hunk 8 above)
Comment (classicalliu) [reply docs/poly-apis.md]: Updated

File: docs/poly-apis.md (hunk 9 above)
Comment (classicalliu) [reply docs/poly-apis.md]: Updated

File: docs/poly-apis.md (hunk 10 above)
Comment (classicalliu) [reply docs/poly-apis.md]: Updated

File: docs/poly-apis.md (hunk 11 above)
Comment (classicalliu) [reply docs/poly-apis.md]: Updated

**Review by keroro520 — APPROVED** [review]

**Review by RetricSu — APPROVED** [review]

**Review by Flouse — APPROVED** [review]

**Pull request merged**